<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Facility Control</string>
  </property>
  <widget class="QGroupBox" name="groupBox_belt_control">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>벨트 수동 제어</string>
   </property>
   <widget class="QPushButton" name="pushButton_belt_start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>561</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>가동 시작</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/play.png</normaloff>../icons/play.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_gate_control">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>게이트 수동 제어</string>
   </property>
   <widget class="QPushButton" name="pushButton_gate_b_open">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>게이트 B 열기</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/open.png</normaloff>../icons/open.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_gate_a_open">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>게이트 A 열기</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/open.png</normaloff>../icons/open.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_facility_status_log">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>951</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>시설 제어 로그</string>
   </property>
   <widget class="QTextEdit" name="textEdit_facilty_status_log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>39</y>
      <width>931</width>
      <height>191</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu Sans'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu';&quot;&gt;시설 제어 로그가 여기에 표시됩니다.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear_log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>300</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>로그 지우기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_refresh_log">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>240</y>
      <width>300</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>로그 새로고침</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_export_log">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>240</y>
      <width>300</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>로그 내보내기</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_silo_control">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>사일로 수동 제어</string>
   </property>
   <widget class="QPushButton" name="pushButton_silo_change_direction">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>LOAD_B로 전환</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_silo_start">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>적재 시작</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_faciltiy_status">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>351</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>시설 상태</string>
   </property>
   <widget class="QPushButton" name="pushButton_refresh_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>상태 새로고침</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_facility_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>331</width>
      <height>171</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu Sans'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;현재 시설 상태가 여기에 표시됩니다.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
